### ISSUE TITLE
It appears that "make doc" still depends on camlp4

### ISSUE THREAD

**Issue by pmetzger** [garrigue/lablgtk#85]
Howdy! The following lines seem to be in the Makefile, and they prevent a build on a machine without camlp4 installed.

```
gtkdoc.cmo : gtkdoc.ml
        $(CAMLC) -I +ocamldoc -pp "$(CAMLP4O) pa_macro.cmo $(ODOC_DEF)" -c $<
```

**Comment by garrigue** [comment]
Indeed, there is no way to install the docs without camlp4 or camlp5.
Note that you can replace camlp4 by camlp5 if the former is not available for your ocaml version.

**Comment by pmetzger** [comment]
Ah. I presume removing that as well would be unpleasant? I do note that the package itself no longer depends on camlp4.

**Comment by ejgallego** [comment]
It think it is very easy to remove the `camlp5` dependency on gtkdoc; the if should not apply anymore.

We could even make it run in gtk3 without too much effort [by writing a custom dune rule]

**Comment by ejgallego** [comment]
See https://github.com/garrigue/lablgtk/pull/96 for a start.
